bin/link_finder.requirements.txt

```diff
@@ -0,0 +1 @@
+selenium==3.4.3
```

Comment:
add keras and numpy too.

Reply:
added another file for them because they are a part of different program

Reply:
why did you never added readme description for them?

Reply:
My apologies. I have added for them now.